modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -3769,6 +3769,27 @@ void GDScriptAnalyzer::reduce_identifier_from_base(GDScriptParser::IdentifierNod
 		}
 	}
 
+	// check non GDScript Scripts
```

Comment:
```suggestion
	// Check non-GDScript scripts.
```
We use sentence case for comments.

Sentence case is like the above comment.

Reply:
Thanks for the feedback.